src/metabase/pulse/send.clj

```diff
@@ -67,6 +68,7 @@
                               :dashboard_id                     (:id dashboard)
                               :parameters                       (:parameters pulse)
                               :skip_if_empty                    (:skip_if_empty pulse)
+                              :disable_links                    (embed.util/modular-embedding-context? (:creation_context pulse))
```

Comment:
This is an intermediate object that will get picked by the function `notification.payload/payload` in `src/metabase/notification/payload/impl/dashboard.clj`